pyrolite/geochem/alteration.py

```diff
@@ -127,3 +127,26 @@ def WIP(df: pd.DataFrame):
 
     """
     return 2 * df.Na2O / 0.35 + df.MgO / 0.9 + 2 * df.K2O / 0.25 + df.CaO / 0.7
+
+def AIOCG_xy(df: pd.DataFrame):
```

Comment:
Also, I think keeping the name to `AIOCG` is appropriate here; the xy aspect is implied in the diagram.